jlab-sensing/ENTS-node-firmware
# Add adc calibration support

---
name: Stephen Taylor
about: calibration eqn for the demo
title: 'Junior Specialist'
labels: ''
assignees: 'jmadden173'
reviewers: 'jmadden173'
---

**Name/Affiliation/Title**
Stephen Taylor, UCSC, staff

**Purpose of the PR**
Added a 'calibration' folder for calibration procedure. Integrated the positive voltage calibration for the purposes of the demo

**Development Environment**
Windows

**Test Procedure**
Run calibrate adc with a soil power sensor and the 2450 and see if it outputs reasonable values.

**Additional Context**
Closes #50
